External/FEXCore/Source/Interface/Core/JIT/Arm64/JITClass.h

```diff
@@ -29,18 +29,28 @@ namespace FEXCore::CPU {
 using namespace vixl;
 using namespace vixl::aarch64;
 
-const std::array<aarch64::Register, 24> RA64 = {
-  x4, x5, x6, x7, x8, x9,
-  x10, x11, x12, x13, x14, x15,
-  x16, x17,
-  x18, x19, x20, x21, x22, x23,
-  x24, x25, x26, x27};
-
-const std::array<std::pair<aarch64::Register, aarch64::Register>, 12> RA64Pair = {{
+//
+const std::array<aarch64::Register, 16> SRA64 = {
+  x19, x20, x21, x22, x23, x24, x25, x26,
+  x27, x29, x18, x17, x16, x15, x14, x13
+};
+
+// RA64 array has more entries for spilling
+#define RA64_COUNT 9
```

Comment:
Modify `RA64` size, this was temporary while spilling was depending on the RA64 logic